homeassistant/components/blink/__init__.py

```diff
@@ -108,6 +99,14 @@ async def async_setup_entry(hass: HomeAssistant, entry: BlinkConfigEntry) -> boo
     return True
 
 
+@callback
+def _async_update_entry_data(
+    hass: HomeAssistant, entry: BlinkConfigEntry, blink: Blink
+) -> None:
+    """Update the config entry data after token refresh."""
+    hass.config_entries.async_update_entry(entry, data=blink.auth.login_attributes)
```

Comment:
Side note: This isn't good. We want to keep control over the config entry data keys in the integration.

Reply:
I can work on it later. What do you suggest?

Reply:
I'd have a function in the integration to create the config entry data from the `blink.auth.login_attributes` since we seem to create the config entry data in more than one place in the integration.